map_model/src/objects/turn.rs

```diff
@@ -360,3 +367,14 @@ fn movement_geom(
     }
     PolyLine::deduping_new(pts)
 }
+
+impl TurnID {
+    pub fn to_movement(self, map: &Map) -> MovementID {
+        MovementID {
+            from: map.get_l(self.src).get_directed_parent(map),
```

Comment:
The lane->directed road mapping is starting to be used frequently now; I'm considering directly storing the ID on `Lane`.

Reply:
by "the ID" do you mean the `DirectedRoadID`?

Alternatively, if I'm reading it correctly, we since we already have the RoadID, we could cheaply synthesize DirectedRoadID if we additionally stored just the Direction rather than the entire DirectedRoadID. No biggie but would save a few bytes per.

Reply:
Ah the Github thread just showed up without my original comment, now I understand.

I see what you mean -- `Lane` can just store `Direction`. Yes, I'll make that change in a followup -- since it requires regenerating all maps, I'd like to bundle it with the next step, which has to do that anyway.